deploy/cluster-roles/blackbox-exporter-clusterrole_binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: blackbox-exporter
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: blackbox-exporter
+subjects:
+  - kind: ServiceAccount
+    name: blackbox-exporter-service-account
+    namespace: redhat-rhoam-middleware-monitoring-operator
```

Comment:
Other CRB files reference `application-monitoring ` namespace, e.g.:
https://github.com/integr8ly/application-monitoring-operator/blob/c6fc97334e4e88eb3f05aef0cc4671a35e242a74/deploy/cluster-roles/alertmanager-clusterrole_binding.yaml#L11
OLM will then take care of updating this field during installation/upgrade.

Reply:
Fixed